UnicodeEncodeError: 'ascii' codec can't encode characters in position 91-92: ordinal not in range(128)

stack:

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/mongo_connector-2.4-py2.7.egg/mongo_connector/util.py", line 90, in wrapped
    func(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/mongo_connector-2.4-py2.7.egg/mongo_connector/oplog_manager.py", line 303, in run
    ns, timestamp)
  File "/usr/local/lib/python2.7/dist-packages/mongo_connector-2.4-py2.7.egg/mongo_connector/util.py", line 43, in wrapped
    reraise(new_type, exc_value, exc_tb)
  File "<string>", line 2, in reraise
UnicodeEncodeError: 'ascii' codec can't encode characters in position 91-92: ordinal not in range(128)
2016-06-29 06:01:56,828 [ERROR] mongo_connector.connector:304 - MongoConnector: OplogThread <OplogThread(Thread-2, stopped 140607599130368)> unexpectedly stopped! Shutting down
```


```
#entry dict:
{u'h': 4678864365021264437L, u'ts': Timestamp(1467011042, 4), u'o': {u'$set': {u'uniqueScreenName': u'\u597d\u4eba0_0', u'screenName': u'\u597d\u4eba0_0'}}, u'v': 2, u'ns': u'jike._users', u'o2': {u'_id': ObjectId('5770cf568993641300a9c69c')}, u'op': u'u'}
```

--- a/mongo_connector/compat.py
+++ b/mongo_connector/compat.py
@@ -22,7 +22,7 @@ if PY3:
 
 else:
     exec("""def reraise(exctype, value, trace=None):
-    raise exctype, str(value), trace""")
+    raise exctype, value, trace""")
 
     def is_string(x):
         return isinstance(x, basestring)

